app/workers/education_form/forms/va0994.rb

```diff
@@ -50,27 +50,39 @@ def applicant_ssn
       @applicant.applicantSocialSecurityNumber
     end
 
+    def new_bank_info?
+      @applicant.bankAccount&.routingNumber.present? ||
+        @applicant.bankAccount&.accountNumber.present? ||
+        @applicant.bankAccount&.accountType.present?
+    end
+
     def bank_routing_number
       if @applicant.bankAccount&.routingNumber.present?
         @applicant.bankAccount.routingNumber
-      else
+      elsif !new_bank_info?
         value_or_na(@applicant.prefillBankAccount&.bankRoutingNumber)
```

Comment:
Since you are testing for the value to  be present multiple times, do you still need to pass it to this `value_or_na` method? Same for lines 73 and 83